chart/values.yaml

```diff
@@ -15,13 +15,13 @@ postgresql:
   # if `enabled = false` you need to set database.host
   enabled: true
   image:
-    tag: 9.6.15
+    tag: latest
```

Comment:
I think this line should be removed altogether, not just set to `latest`. Using `latest` would definitely not be optimal. What we could do is update the actual chart version we are pulling in to the latest release.

That being said, I would like to remove these external charts from this chart altogether.